tests/vectordb/test_chroma_db.py

```diff
@@ -18,15 +16,10 @@ def test_init_with_host_and_port(self):
         host = "test-host"
         port = "1234"
 
-        with patch.object(chromadb, "HttpClient") as mock_client:
-            _db = ChromaDB(host=host, port=port, embedding_fn=len)
-
-        expected_settings = Settings(
-            chroma_server_host="test-host",
-            chroma_server_http_port="1234",
-        )
-
-        mock_client.assert_called_once_with(expected_settings)
```

Comment:
Mocking (usually with stubbing) is generally when depending on external systems. I think the reason this test didn't pass is because the mocking object was not passed into ChromeDB. Changed it to simpler assertion statements.